modules/KalturaSupport/components/chapters/chapters.js

```diff
@@ -462,8 +462,12 @@
 
 			//Get current item number
 			var orderId = this.mediaList.length;
+			
+			var clearSameCP = items.filter(function( item,index,allInArray ) {
```

Comment:
@kalturaguy 
Do you think we should optimize this? From how it is it looks like we run this thing with EVERY CP that is processed in the logic

Reply:
@guy1976 we checked and this happens on CP load and only once, so we are good here